unified-build buildpass=2 job is broken

Regressed with https://github.com/dotnet/sdk/commit/882392768222def8420266abbca8c81a40322ce1

https://dev.azure.com/dnceng/internal/_build/results?buildId=2631403&view=results

> D:\a\_work\1\s\src\sdk\eng\Signing.props(4,5): error MSB4184: The expression "[System.IO.File]::ReadAllText(D:\a\_work\1\s\src\sdk\artifacts\tmp\Release\FullNugetVersion.version)" cannot be evaluated. Could not find file 'D:\a\_work\1\s\src\sdk\artifacts\tmp\Release\FullNugetVersion.version'. [D:\a\_work\1\s\artifacts\source-built-sdks\Microsoft.DotNet.Arcade.Sdk\tools\Publish.proj]

--- a/eng/Signing.props
+++ b/eng/Signing.props
@@ -1,8 +1,8 @@
 <Project>
 
   <PropertyGroup>
-    <BlobStorageVersion>$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)FullNugetVersion.version').Trim())</BlobStorageVersion>
-    <RelativeBlobPathParent>Sdk/$(BlobStorageVersion)</RelativeBlobPathParent>
+    <!-- The FullNugetVersion.version file only gets produced in the initial build pass. -->
+    <RelativeBlobPathParent Condition="'$(DotNetBuildPass)' == '' or '$(DotNetBuildPass)' == '1'">Sdk/$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)FullNugetVersion.version').Trim())</RelativeBlobPathParent>
     <PublishBinariesAndBadge Condition="'$(PublishBinariesAndBadge)' == ''">true</PublishBinariesAndBadge>
   </PropertyGroup>
 

